src/weights.jl

```diff
@@ -569,47 +565,62 @@ function quantile(v::RealVector{V}, w::AbstractWeights{W}, p::RealVector) where
         x < 0 && error("weight vector cannot contain negative entries")
     end
 
-    # full sort
-    vw = sort!(collect(zip(v, w.values)))
+    wvalues = w.values
+    nz = find(w.values)
+    #normalize if non frequencyweight
```

Comment:
Space after # (also below) and between "frequency" and "weights". Better also say that the sum will be 1.